src/app/layout/footer/footer.component.html

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <div class="footer-nav">
+    <a class="footer-link" href="#">Admin</a>
+    &middot;
+    <a class="footer-link" href="#">Librarian</a>
+    &middot;
+    <a class="footer-link" href="#">Borrower</a>
+  </div>
```

Comment:
These hrefs should become routerLinks once they're made to point to the components, so the app stays on the same page instead of reloading.